code_search/demo/README.md

```diff
@@ -3,4 +3,32 @@
 This directory contains assets for setting up a demo of the code search example.
 It is primarily intended for use by Kubeflow contributors working on the shared demo.
 
-Users looking to run the example should follow the README.md in the parent directory.
\ No newline at end of file
+Users looking to run the example should follow the README.md in the parent directory.
+
+# GCP Resources
+
+We are using the following project
+
+* **org**: kubeflow.org
+* **project**: code-search-demo
+* **[code-search-team@kubeflow.org](https://github.com/kubeflow/internal-acls/blob/master/code-search-team.members.txt)** Google group administering access
+
+# Results
+
+## 2018-11-05
+
+jlewi@ ran experiments that produced the following results
+
+| What | location | Description
```

Comment:
Table doesn't render correctly. I think you also need a vertical bar on the right side.